### DISCUSSION TITLE
Offline tracking only 24 hours is not enough

### DISCUSSION THREAD

**Discussion by suntri-star** [osmandapp/OsmAnd#17644] [Ideas]
I use offline tracking, but I'm usually on the road for more than 24 hours without GSM reception. An extension to 30 days would make me very happy. My smartphone certainly has enough space for it.

Offline tracking nur 24 Stunden reichen nicht
Ich nutze das Offline Tracking, bin jedoch meist länger als 24 Stunden ohne GSM empfang unterwegs. Eine erweiterung bis 30 Tage würde mich sehr freuen. Mein Smartphone hat sicher genug Platz dazu.

**Comment by DmitryAlexei** [comment]
You may need to turn off **Auto-split recordings after gap** in Trip recording settings

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/5eed24de-1732-4c89-b182-c8d85ecf9d2b"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by suntri-star** [comment]
I mean the recording of the data when there is no network. Then OSMand saves the data locally. As soon as GSM or WLAN is available, it will be sent. Brilliant. However, I travel a lot in areas without GSM. So the device would have to save longer than 24 hours. That would be awesome.

Ich meine die Aufzeichnung der Daten wenn kein Netz vorhanden ist. Dann speichert OSMand die Daten lokal. Sobald GSM oder WLAN vorhanden ist wird gesendet. Genial. Ich bin jedoch viel in Gebieten ohne GSM unterwegs. Also müsste das Gerät länger als 24h speichern. Das währe der Hammer.

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/59803990/7025d3e3-9330-4829-99ed-384b1bc529f8"  width="300" height="auto"/> </th>
  </tr>
</table>
